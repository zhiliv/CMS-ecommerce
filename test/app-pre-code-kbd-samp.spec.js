import { shallowMount } from '@vue/test-utils'
import OL from '@/components/app/pre-code-kbd-samp/pre-code-kbd-samp.vue'

describe('app-pre-code-kbd-samp.js', () => {
  let cmp

  beforeEach(() => {
    cmp = shallowMount(OL, {
      // Создать поверхностный экземпляр компонента
      propsData: {
        classes: 'simple simple-xl',
        styles: 'background: white',
        id: 'indent',
        type: 'code'
      },

      slots:{
        default: '<li>Тест</li>'
      }
    })
  })
  it('Содержимое html для app-ol-ul ', () => {
    expect(cmp.element).toMatchSnapshot()
  })
})
