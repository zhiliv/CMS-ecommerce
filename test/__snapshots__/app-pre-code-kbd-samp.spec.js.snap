// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app-pre-code-kbd-samp.js Содержимое html для app-ol-ul  1`] = `
<code
  class="simple simple-xl"
  id="indent"
  style="background: white"
>
  <li>
    Тест
  </li>
</code>
`;
